### PR TITLE
Release 2022-11-15

### PR DESCRIPTION
**Pull request by noatamir**
Merge `develop` into `main`.

### CODE REVIEW

File: apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+### Maintain CuPy to continue supporting the latest platforms
+As with other GPU applications, CuPy heavily relies on NVIDIA CUDA and AMD ROCm libraries. Currently, CuPy supports with CUDA versions (10.2, 11.0-11.6) and four ROCm versions (4.0, 4.2, 4.3, 5.0). GPU vendors generally release updates of these toolkits on a quarterly basis, and providing immediate support for them is essential to allow users to enjoy the latest features, performance enhancements, security, and bug fixes. This work includes:
+
+* Implementing Python wrappers for new APIs added to the library to make them available to CuPy
+* Supporting  new features from a new CUDA or ROCm release, when applicable 
+* Setting up a continuous integration system to provide extensive coverage of the supported software combinations
+* Building a delivery pipeline to provide binary packages for the supported versions of the library
+
+
+# Want to join?
```
Comment (noatamir) [inline apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md:105]:
```suggestion
<br />

# Want to join?
```

File: apps/labs/posts/lets-make-pygments-accessible.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
+title: '"Making pygments accessible"'
```
Comment (noatamir) [inline apps/labs/posts/lets-make-pygments-accessible.md:2]:
```suggestion
title: 'Making pygments accessible'
```

**Review by bskinn — COMMENTED** [review]
Consulting content CHANGELOG edits LGTM.

Leaving this as 'Comment' instead of 'Approve' since there are hotfix edits happening to the Labs blog post. 

Will need to undelete the release branch after merge to `main` so that we can back-merge these hotfixes into `develop`. (Then re-delete.)

**Review by trallard — APPROVED** [review]

**Comment by noatamir** [comment]
@bskinn if you rather I undo those changes and make them on a clean PR I can. What would be simplest?

**Comment by bskinn** [comment]
No, the back-merge of the hotfixes to `develop` is fine with me -- @gabalafou and I have done that before.

Just wanted to make note of the need of the back-merge.

**Comment by bskinn** [comment]
(At least, for minor edits like this, the back-merge is fine w/me. I don't know if there's been discussion on how we want to handle it if we discover a need for a major change while a release is in progress.)

**Pull request merged**